keyboards/gon/nerdtkl/keymaps/gam3cat/config.h

```diff
@@ -1,8 +1,27 @@
+/*Copyright 2021 @Gam3cat
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.*/
+
 #pragma once
 
 //Force NKRO to be enabled.
 #define FORCE_NKRO
 
+//Force polling rate.
+#define USB_POLLING_INTERVAL_MS 1
+#define QMK_KEYS_PER_SCAN 12
```

Comment:
Since you're adding these to all of the keymaps, you could move this to `/users/gam3cat/config.h` instead.

Reply:
Updated